frame/nfts/src/features/settings.rs

```diff
@@ -0,0 +1,60 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::*;
+use frame_support::pallet_prelude::*;
+
+impl<T: Config<I>, I: 'static> Pallet<T, I> {
```

Comment:
docs here could give an example of a couple of likely nft settings so that people get the idea.

Reply:
done

Reply:
ah so these are part of the feature machinery rather than being a separate settings feature. Wondering if we should one module for features and another for the mechanism.

Reply:
@gilescope where would you put them?

Reply:
Here's fine for now. We can revisit locations when we have another pallet that needs the same code.